src/components/Media/Media.tsx

```diff
@@ -5,17 +5,21 @@ import MediaCaption from './MediaCaption';
 
 type MediaProps = {
   caption?: string;
+  captionVariant?: 'below' | 'right';
   children: React.ReactElement;
   className?: string;
   credit?: string;
+  hideCaption?: boolean;
   licence?: string;
 };
 
 const Media = ({
   caption,
+  captionVariant = 'right',
```

Comment:
I decided to use `right` as a default as this is our default for images and videos in CR.

Reply:
Good consistency